### ISSUE TITLE
Keyboard accessible knob/range controls

### ISSUE THREAD

**Issue by theriex** [theriex/digger#31]
It should be possible to tab through the Digger control surface elements, then use the the arrow keys to adjust values.  Unless there is some accessibility overlay that obviates this need.
